src/Components/Comments.jsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import { fetchData } from '../Utils/api';
+
+const Comments =({article_id}) => {
+
+    const [comments, setComments] = useState([])
+
+    useEffect(() => {
+        fetchData(`/articles/${article_id}/comments`).then(({comments})=>{
+            setComments(comments)
+        }).catch((err)=>{
+            console.log(err);
+        })
```

Comment:
Using fetch is OK at this stage, but axios will tell the what the error is (if there's an error). Will be easier to do error handling if it's converted to axios.